### PR TITLE
Prepare CI for Python 3.12 minimal container image

### PR DESCRIPTION
**Pull request by frenzymadness**
When we merge this, I'll rebase #704

### CODE REVIEW

**Comment by frenzymadness** [comment]
@phracek ping

**Comment by phracek** [comment]
[test]

File: .github/workflows/container-tests.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [ "3.8", "3.9", "3.9-minimal", "3.11", "3.11-minimal", "3.12" ]
+        version: [ "3.8", "3.9", "3.9-minimal", "3.11", "3.11-minimal", "3.12", "3.12-minimal" ]
```
Comment (phracek) [inline .github/workflows/container-tests.yml:63]: In order to test it, this change has to be done as a separate commit.

**Review by phracek — CHANGES_REQUESTED** [review]
Please address my comment.

**Review by phracek — APPROVED** [review]
Sorry, I was on vacation. LGTM.

**Pull request merged**